packages/wrangler/src/r2/helpers.ts

```diff
@@ -400,23 +404,25 @@ export async function putEventNotificationConfig(
 		`Sending this configuration to "${bucketName}":\n${JSON.stringify(body)}`
 	);
 	return await fetchResult<{ event_notification_detail_id: string }>(
-		`/accounts/${accountId}/event_notifications/r2/${bucketName}/configuration/queues/${queueUUID}`,
+		`/accounts/${accountId}/event_notifications/r2/${bucketName}/configuration/queues/${queue.queue_id}`,
 		{ method: "PUT", body: JSON.stringify(body), headers }
 	);
 }
 
 export async function deleteEventNotificationConfig(
+	config: Config,
 	apiCredentials: ApiCredentials,
 	accountId: string,
 	bucketName: string,
-	queueUUID: string
+	queueName: string
 ): Promise<null> {
+	const queue = await getQueue(config, queueName);
```

Comment:
Well, in terms of API surface, the key change is using queue name instead of queue ID. But under the hood, this is the piece that makes that change possible.